plantuml/LinkLoss/data_written_normal.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+participant LinkLossService as LLS
+participant Gap as GAP
```

Comment:
I would model the emitter of the `onDataWritten` call as the BLE peer or the BT stack.